interview-questions.md

```diff
@@ -8,50 +8,52 @@ Then, research the question to expand on your answer. Even if you feel you have
 
 1. What is the difference between a parameter and an argument?
 
-Your answer:
+Your answer: A parameter is the custom name of data that you will referenece in a function, it doesnt matter what you call it, the function will still work as it's only a placeholder name. However it's best to call it something meaningful and accurate, for looking back on your own code, or for when others read it. An argument is the actual data you put into the function 
 
-Researched answer:
+Researched answer: A variable that lives within the function definition, it is the name for data that will go through that function. An argument is data that will be passed into the function and will be called on.
 
 2. The JavaScript built in method .map() takes predefined parameters. What are they? Which are required and which are optional?
 
-Your answer:
+Your answer: It can only be used on an array, it iterates through and gives you an array the same size as the one you put in. I'm unsure of the optional parameter. (That'
+s everything I know about .map)
 
-Researched answer:
+Researched answer: I can't find much about. map predefined parameters. I found that .map does not require a parameter in the function and can use an anon function. 
```

Comment:
The parameters would be value, index, and array, with only value being required.